wing-redis/redis.sim.w

```diff
@@ -0,0 +1,88 @@
+bring "./api.w" as api;
+bring "./utils.w" as utils;
+bring cloud;
+bring util;
+bring sim;
+
+pub class Redis_sim impl api.IRedis {
+    redisContainerName: str;
+    redisImage: str;
+    state: sim.State;
+
+    init() {
+    this.redisContainerName = "wing-sim-redis-${this.node.id}-${util.uuidv4()}";
+    // Redis version 7.0.9
+    this.redisImage = "redis@sha256:e50c7e23f79ae81351beacb20e004720d4bed657415e68c2b1a2b5557c075ce0";
+    this.state = new sim.State();
+    // start and stop the instance
+    let s = new cloud.Service(inflight () => {
```

Comment:
Please use `containers.Workload` instead of hand rolling. 

See the Postgres library as an example.